app/components/previews/provider_interface/volunteering_history_component_preview.rb

```diff
@@ -0,0 +1,17 @@
+module ProviderInterface
+  class VolunteeringHistoryComponentPreview < ActionView::Component::Preview
+    def last_application
```

Comment:
In other components we've written the preview so that it picks a random model from a collection of models that match the use case. See e.g. https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/app/components/previews/provider_interface/status_box_component_preview.rb#L53

This helps preview the component with a range of data, e.g. by refreshing the preview page. Is it worth doing this here, too?

Reply:
I've changed the logic in the previews to select a 'random' application from the set of 'qualifying' applications (those that have the right combination of child objects - work experience, volunteering experience and work history breaks). Works with https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1578